Description: small typo in FormBuilder.pod
 alot -> a lot
Author: Damyan Ivanov <dmn@debian.org>
Forwarded: no
Last-Update: 2019-01-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/CGI/FormBuilder.pod
+++ b/lib/CGI/FormBuilder.pod
@@ -2056,7 +2056,7 @@
         font-style: italic;
     }
 
-Of course, if you're familiar with CSS, you know alot more is possible.
+Of course, if you're familiar with CSS, you know a lot more is possible.
 Also, you can mess with all the id's (if you name your forms) to
 manipulate fields more exactly.
 
